integration_tests/tests/billing-and-transaction-view-tests.ts

```diff
@@ -9,56 +11,47 @@ import {
 } from "../../src/handlers/int-test-support/helpers/commonHelpers";
 import {
   deleteS3Objects,
-  putS3Object,
-  S3Object,
   listS3Objects,
 } from "../../src/handlers/int-test-support/helpers/s3Helper";
-import path from "path";
-import fs from "fs";
 import {
   EventName,
   VendorId,
 } from "../../src/handlers/int-test-support/helpers/payloadHelper";
 import { queryResponseFilterByVendorServiceNameYear } from "../../src/handlers/int-test-support/helpers/queryHelper";
+import {
+  randomLineItem,
+  randomInvoice,
+} from "../../src/handlers/int-test-support/helpers/mock-data/invoice/random";
+import { getVendorServiceConfigRow } from "../../src/shared/utils/config-utils";
+import { createInvoiceInS3 } from "../../src/handlers/int-test-support/helpers/mock-data/invoice/helpers";
 
 const prefix = resourcePrefix();
 const bucketName = `${prefix}-storage`;
+const testStartTime = new Date();
 
 describe("\nUpload invoice to standardised folder and verify billing and transaction_curated view query results matches with expected data \n", () => {
-  const folderPrefix = "btm_billing_standardised";
-  const testObject: S3Object = {
-    bucket: `${prefix}-storage`,
-    key: `${folderPrefix}/receipt.txt`,
-  };
-  beforeAll(async () => {
+  beforeEach(async () => {
     // tests are enabled to run sequentially as we are deleting the S3 directory in view tests so when running the test
     // in parallel other tests will be interrupted(e.g. sns-s3 tests generate and checks eventId). We can enable to run in parallel
     // once we implement BTM-340 to clean up after each test
```

Comment:
I can't find BTM-340 in Jira so I don't remember what it is was for and if these comments need to be updated.

Reply:
this is the ticket to improve the cleanup strategy https://govukverify.atlassian.net/browse/BTM-340